faucet/vlan.py

```diff
@@ -271,15 +273,15 @@ def reset_ports(self, ports):
         self.dot1x_untagged = tuple([port for port in sorted_ports
                                      if self == port.dyn_dot1x_native_vlan])
 
-    def add_cache_host(self, eth_src, port, cache_time):
+    def add_cache_host(self, eth_src, port, cache_time, l3_src_ip=None):
```

Comment:
why default?

Reply:
add_cache_host is not being used anywhere else, afaik, but just leaving it as default to make sure previous signature works as well.

Reply:
but why?  As per other comment, better to remove it.